data/releasenotes.json

```
@@ -1,5 +1,54 @@
+			"<b>PostgreSQL compatibility:</b> Stubbed <tt>pg_partition_ancestor</tt> system table.",
+            "<b>PostgreSQL compatibility:</b> Added support for <tt>with ordinality</tt> in <tt>pg_partition_ancestor</tt> system table.",
+			"<b>PostgreSQL compatibility:</b> Added support for <tt>with ordinality</tt> in <tt>array_unnest()</tt> function.",
+			"<b>PostgreSQL compatibility:</b> Server now correctly reports compatibility with PostgreSQL 16.1 instead of 11.3.",
+			"<b>PostgreSQL compatibility:</b> Get rid of the custom namespace <tt>cedar</tt> which caused issues with some tools.",
+            "<b>PostgreSQL compatibility:</b> Allow casts from oids inside strings.",
+			"<b>PostgreSQL compatibility:</b> CedarDB no longer prints implicitly created foreign key indexes in <tt>pg_class</tt> and <tt>pg_index</tt>.",
+			"<b>PostgreSQL compatibility:</b> Print the full name when calling <tt>pg_get_indexdef()</tt> on a table that is not in the search path.",
+			"<b>PostgreSQL compatibility:</b> <tt>pg_table_is_visible</tt> now works on indexes and sequences.",
+			"<b>PostgreSQL compatibility:</b> Use Postgres's special legacy <tt>oidvector</tt> and <tt>int2vector</tt> types in <tt>pg_index</tt>.",
+			"<b>PostgreSQL compatibility:</b> CedarDB now correctly enforces unique names across tables, views, indexes, and sequences. <emph>If you previously two of those objects with the same name, they have been renamed!</emph>. So if you had a table and a sequence both called <tt>foo</tt>, the sequence is now called <tt>foo1</tt>.",
+			"<b>PostgreSQL compatibility:</b> <tt>SELECT * FROM <sequence></tt> now works and returns its last value, the number of reserved sequence values and whether it was already called.",
+			"<b>PostgreSQL compatibility:</b> Sequence limits now behave identical to Postgres.",
+			"<b>PostgreSQL compatibility:</b> Implement <tt>pg_opclass</tt> system table.",
+			"<b>Performance:</b> Streamlined counting characters with SSE.",
+			"<b>Performance:</b> Added support for AES hardware accelerated hashing via gxhash.",
+			"<b>Performance:</b> Expressions of the type <tt>regex_replace(in, pattern, '\\1')</tt> are no optimized to <tt>regexp_substr()</tt>.",
+			"<b>Performance:</b> Leading and trailing wildcards in <tt>regexp_substr()</tt> are now dropped.",
+			"<b>Performance:</b> CedarDB now accepts comments which contain invalid UTF-8.",
+			"<b>Maintenance:</b> Updated ICU version to 76.1, introducing support for new unicode characters.",
+			"<b>Maintenance:</b> Fixed off-by-one error in <tt>regexp_substr()</tt>.",
+			"<b>Maintenance:</b> Ensure sequences now have an explicit start value.",
+			"<b>Maintenance:</b> Severely hardened UTF-8 validation.",
+			"<b>Maintenance:</b> Fixed on OOB access when parsing ISO8601 intervals which lead to a crash.",
+			"<b>Maintenance:</b> Fixed a bug where generate_series() didn't correctly work when used in conjunction with arrays that weren't 1-indexed.",
```

```suggestion
			"<b>Maintenance:</b> Fixed a bug where generate_subscripts() didn't correctly work when used in conjunction with arrays that weren't 1-indexed.",
```
I assume this is subscripts?

---

No